build/tools/Automation/Python/Python.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Oryx.Automation.Client;
+using Microsoft.Oryx.Automation.Models;
+using Microsoft.Oryx.Automation.Python.Models;
+using Microsoft.Oryx.Automation.Services;
+using Microsoft.Oryx.Automation.Telemetry;
+using Newtonsoft.Json;
+using Oryx.Microsoft.Automation.Python;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Microsoft.Oryx.Automation.Python
+{
+    public class Python
+    {
+        private readonly IHttpClient httpClient;
+        private readonly ILogger logger;
+        private readonly IVersionService versionService;
+        private readonly IYamlFileReaderService yamlFileReaderService;
+
+        public Python(IHttpClient httpClient, ILogger logger, IVersionService versionService, IYamlFileReaderService yamlFileReaderService)
```

Comment:
nit: would you mind moving each argument to a different line?

```cs
public Python(
    IHttpClient httpClient,
    ILogger logger,
    IVersionService versionService,
    IYamlFileReaderService yamlFileReaderService)
```

Reply:
Updated